qiskit_ibm_experiment/client/local_client.py

```diff
@@ -0,0 +1,534 @@
+        experiment_type: Optional[str] = None,
+        backend_name: Optional[str] = None,
+        tags: Optional[List[str]] = None,
+        parent_id: Optional[str] = None,
+        tags_operator: Optional[str] = "OR",
+        **filters: Any,
+    ) -> str:
+        """Retrieve experiments, with optional filtering.
+
+        Args:
+            limit: Number of experiments to retrieve.
+            marker: Marker used to indicate where to start the next query.
+            backend_name: Name of the backend.
+            experiment_type: Experiment type.
+            start_time: A list of timestamps used to filter by experiment start time.
+            device_components: A list of device components used for filtering.
+            tags: Tags used for filtering.
+            hub: Filter by hub.
+            group: Filter by hub and group.
+            project: Filter by hub, group, and project.
+            exclude_public: Whether or not to exclude experiments with a public share level.
+            public_only: Whether or not to only return experiments with a public share level.
+            exclude_mine: Whether or not to exclude experiments where I am the owner.
+            mine_only: Whether or not to only return experiments where I am the owner.
+            parent_id: Filter by parent experiment ID.
+            sort_by: Sorting order.
+
+        Returns:
+            A list of experiments and the marker, if applicable.
+        """
+        df = self._experiments
+
+        if experiment_type is not None:
+            if experiment_type[:5] == 'like:':
+                experiment_type = experiment_type.split(":")[1]
+                df = df.loc[df.type.str.contains(experiment_type)]
+            else:
+                df = df.loc[df.type == experiment_type]
+
+        if backend_name is not None:
+            df = df.loc[df.device_name == backend_name]
+
+        # Note a bug in the interface for all services:
+        # It is impossible to filter by experiments whose parent id is None
+        # (i.e., root experiments)
+        if parent_id is not None:
+            df = df.loc[df.parent_experiment_uuid == parent_id]
+
+        # Waiting for consistency between provider service and qiskit-experiments service,
+        # currently they have different types for `device_components`
+        if device_components is not None:
+            raise ValueError(
+                "The fake service currently does not support filtering on device components"
+            )
+
+        if tags is not None:
+            if tags_operator == "OR":
+                df = df.loc[df.tags.apply(lambda dftags: any(x in dftags for x in tags))]
+            elif tags_operator == "AND":
+                df = df.loc[df.tags.apply(lambda dftags: all(x in dftags for x in tags))]
+            else:
+                raise ValueError("Unrecognized tags operator")
+
+        # These are parameters of IBMExperimentService.experiments
```

Comment:
This comment and other similar comments in this methods, some of them I guess are to be updated

Reply:
Agree, we don't need them anymore.